ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -167,6 +173,15 @@ tests:
       SOURCE_IMAGE_REF: assisted-service
     test:
     - ref: assisted-baremetal-images-publish
+- as: mirror-nightly-image-el8
```

Comment:
no need for a build after a commit is merged?

Reply:
Ah, missed the second job.